plugin/pkg/scheduler/scheduler.go

```diff
@@ -222,7 +225,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	if err != nil {
```

Comment:
the call to `FinishBinding` was moved in this PR: https://github.com/kubernetes/kubernetes/pull/48451 -- let me know if you think it should be moved back (and why? :) ).